src/ResourceManager/Compute/Commands.Compute/Models/PSVirtualMachineDataDisk.cs

```diff
@@ -0,0 +1,29 @@
+// 
+// Copyright (c) Microsoft and contributors.  All rights reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//   http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// 
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+// Warning: This code was generated by a tool.
+// 
+// Changes to this file may cause incorrect behavior and will be lost if the
+// code is regenerated.
+
+using Microsoft.Azure.Management.Compute.Models;
+
+namespace Microsoft.Azure.Commands.Compute.Models
+{
+    public class PSVirtualMachineDataDisk : DataDisk
```

Comment:
Is this class ever going to add any new functionality over the class _DataDisk_?

Reply:
OneSDK asked to convert any client library object (in this case, Management.Compute.Models) to a powershell object (in this case, Commands.Compute.Models) for a return object, so that it is easy to hide a possible breaking change (in the future) caused by a client library.